tin/apps/assignments/models.py

```diff
@@ -172,6 +173,16 @@ class Assignment(models.Model):
         validators=[MinValueValidator(10)],
     )
 
+    use_submission_cap = models.BooleanField(default=False)
```

Comment:
Would be nice to enable/disable each cap independently. What if I just want a cap of 0 after the due date, but no total cap?